apps/contracts/hardhat.config.ts

```diff
@@ -9,7 +9,7 @@ import 'hardhat-contract-sizer'
 import 'solidity-coverage'
 import 'tsconfig-paths/register'
 
-import { HardhatUserConfig } from 'hardhat/types'
+import { HardhatUserConfig } from 'node_modules/hardhat/types'
```

Comment:
Why is this change needed? It seems to run on my machine with the previous version.

Reply:
@jtse0 ping!

Reply:
I've checked this issue and it seems like it should work with the previous versions so I have reverted this accordingly.  Updated in the latest changes.